content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -0,0 +1,547 @@
+$ dvc exp show --exclude-params train.epochs --exclude-metrics auc
+````
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
+┃ neutral:**Experiment**            ┃ neutral:**Created**      ┃    metric:**acc** ┃ param:**model.conv_units** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
+│ workspace             │ -            │ 0.9127 │ 16               │
+│ baseline-experiment   │ Sep 06, 2021 │ 0.9127 │ 16               │
+│ ├── ca55d7d [cnn-32]  │ Sep 09, 2021 │ 0.9167 │ 32               │
+│ ├── 6d13f33 [cnn-64]  │ Sep 09, 2021 │ 0.9153 │ 64               │
+│ ├── 49779aa [cnn-96]  │ Sep 09, 2021 │ 0.9161 │ 96               │
+│ └── 69503c6 [cnn-128] │ Sep 09, 2021 │  0.916 │ 128              │
+└───────────────────────┴──────────────┴────────┴──────────────────┘
+```
+
+Another column that appear in `dvc exp show` table is the timestamp column. You
+may want to hide it with `--no-timestamp` option.
+
+```dvc
+$ dvc exp show --no-timestamp --include-params=model.conv_units --exclude-metrics=loss
+```
+
+````dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
+┃ neutral:**Experiment**            ┃    metric:**acc** ┃ param:**model.conv_units** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
+│ workspace             │ 0.9127 │ 16               │
+│ baseline-experiment   │ 0.9127 │ 16               │
+│ ├── ca55d7d [cnn-32]  │ 0.9167 │ 32               │
+│ ├── 6d13f33 [cnn-64]  │ 0.9153 │ 64               │
+│ ├── 49779aa [cnn-96]  │ 0.9161 │ 96               │
+│ └── 69503c6 [cnn-128] │  0.916 │ 128              │
+└───────────────────────┴────────┴──────────────────┘
+
+By default `dvc exp show` lists the experiments by the timestamp. You can sort
+the columns by params or metrics by the option `--sort-by` and `--sort-order`.
+`--sort-by` takes a metric or parameter name as defined in `dvc.yaml` and
+`--sort-order` takes either `asc` or `desc`.
+
+```dvc
+$ dvc exp show --sort-by auc --sort-order desc
+````
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
+┃ neutral:**Experiment**            ┃    metric:**acc** ┃ param:**model.conv_units** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
+│ workspace             │ 0.9127 │ 16               │
+│ baseline-experiment   │ 0.9127 │ 16               │
+│ ├── ca55d7d [cnn-32]  │ 0.9167 │ 32               │
+│ ├── 6d13f33 [cnn-64]  │ 0.9153 │ 64               │
+│ ├── 49779aa [cnn-96]  │ 0.9161 │ 96               │
+│ └── 69503c6 [cnn-128] │  0.916 │ 128              │
+└───────────────────────┴────────┴──────────────────┘
+```
+
+## Get experiments table in JSON
+
+There may be times when a machine readable format for the experiments is
+required. In its default settings `dvc exp show` lists the experiments in a
+visually appealing way that may not be suitable to parse in scripts. To get a
+list of experiments with their params and metrics, you can use `--show-json` or
+`--show-csv` flags.
```

Comment:
It'll be `--json` and `--cvs` soon (as hinted in https://github.com/iterative/dvc.org/pull/2862#issuecomment-932408692) ☝️